tactile_merger/CMakeLists.txt

```diff
@@ -23,26 +25,43 @@ find_package(Eigen3 REQUIRED)
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-#  INCLUDE_DIRS include
+  INCLUDE_DIRS include
 #  LIBRARIES agni_tf_tools
 #  CATKIN_DEPENDS other_catkin_pkg
 #  DEPENDS system_lib
+#  LIBRARIES ${PROJECT_NAME}
```

Comment:
the lib should be exported there, (activate + add lib)

Reply:
my bad, this is for dependent library, you are correct

Reply:
I think `LIBRARIES` should be defined, because that's the library we export.

Reply:
## LIBRARIES: libraries you create in this project that dependent projects also need 
this is EXACTLY what I thought at first, hence my comment, but then read it differently somehow with Jakub, thinking this is to tell which third party libs are also needed and should be exported. But in fact at best I should have put there tactile_filters, since it is not a system (DEPENDS) dependency that catkin can resolve or a catkin (CATKIN_DEPENDS).
Were should tactile_filters go only find_package? 
What about eigen3 ?